spec/features/reward_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+# frozen_string_literal: true
+RSpec.describe 'Reward spec', type: :feature do
+  let!(:reward) { create(:reward) }
+  let!(:employee) { create(:employee) }
+
+  before do
+    login_as employee, scope: :employee
+    visit root_path
+  end
+
+  it 'index and show rewards for employee user' do
+    click_link 'Rewards'
+    expect(page).to have_content 'Rewards Title Description Price'
```

Comment:
what are you checking here exactly? for me it looks like the headers of the Rewards index table. I think we should be checking if `reward.title`, `reward.price` and `reward.description` are being displayed here

Reply:
yes, we should check `reward.title`, `reward.price` and `reward.description` -> corrected